src/mdx/expandVersionedUrl.spec.js

```diff
@@ -21,6 +21,7 @@ describe("expandVersionedUrl", () => {
         "/Users/monkeypants/camunda-platform-docs/optimize/what-is-optimize.md",
         "/docs/next/some/thing",
       ],
+
       [
         "/Users/monkeypants/camunda-platform-docs/optimize_versioned_docs/version-3.8.0/what-is-optimize.md",
```

Comment:
These tests *will* fail when we cut a new version. I think that's fine for now -- we don't actually run them as part of CI anyway, and it's less important to me that the tests are robust and more important that versioning is easier.